Makefile.toml

```diff
@@ -116,6 +116,9 @@ DOCKER_BUILDKIT = "1"
 # write AMI information to specifically named files.
 AMI_DATA_FILE_SUFFIX = "amis.json"
 
+# The type of testsys test that should be run. `integ` will run all available tests.
+TESTSYS_TEST = "quick"
```

Comment:
A description of all the different possible values you can have here would be nice.